mapie/time_series_regression.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import annotations
+
+from typing import Optional, Union
+
+import numpy as np
+from sklearn.base import RegressorMixin
+from sklearn.model_selection import BaseCrossValidator
+
+from .regression import MapieRegressor
+from ._typing import ArrayLike
+
+
+class MapieTimeSeriesRegressor(MapieRegressor):
+    """
+    Prediction interval with out-of-fold residuals for time series.
+
+    This class implements the EnbPI strategy and some variations
+    for estimating prediction intervals on single-output time series.
+    It is ``MapieReegressor`` with one more method ``partial_fit``.
+    Actually, EnbPI only corresponds to MapieRegressor if the ``cv`` argument
+    if of type ``Subsample`` (Jackknife+-after-Bootstrap method). Moreover, for
+    the moment we consider the absolute values of the residuals of the model,
+    and consequently the prediction intervals are symetryc.
```

Comment:
Also add that the `beta` parameter is not included yet ?

Reply:
Done: je ne parle pas de beta, parce qu'il faudrait l'introduire, mais d'optimisation des PIs